app/src/utils/proving/validateDocument.ts

```
@@ -239,3 +246,77 @@ function formatCSCAPem(cscaPem: string): string {
+      if (!authorityKeyIdentifier) continue;
+      await useProtocolStore
+        .getState()
+        [documentCategory].fetch_all(environment, authorityKeyIdentifier);
+      const passportDataAndSecret = await loadPassportDataAndSecret();
+      if (!passportDataAndSecret) continue;
+      const { secret } = JSON.parse(passportDataAndSecret);
+      const isRegistered = await isUserRegistered(migratedPassportData, secret);
+      if (isRegistered) return true;
+    } catch (error) {
+      console.error(`Error in hasAnyValidRegisteredDocument: ${error}`);
+      continue;
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)